CHANGELOG.latest.md

```diff
@@ -1,2 +1,7 @@
-- Introduced New AdServices Integration on iOS. Call `EnableAdServicesAttributionTokenCollection()` to enable this integration.
-More information [in our docs](https://docs.revenuecat.com/docs/apple-search-ads) and in the [announcement blogpost](https://www.revenuecat.com/blog/iad-vs-adservices-whats-the-difference/).
+### API Changes
+* `StoreTransaction`: `RevenueCatId` and `ProductId` have been deprecated in favor of `TransactionIdentifier` and `ProductIdentifier` respectively.
```

Comment:
when did these happen? There's no link to the PR

Reply:
Right, the PR is still open and I added this manually: https://github.com/RevenueCat/purchases-unity/pull/145. Will add the link to the PR